static/js/components/Analytics.js

```diff
@@ -0,0 +1,26 @@
+/* global SETTINGS:false */
+// @flow
+import React from "react"
+import { withRouter } from "react-router"
+import ReactGA from 'react-ga'
+
+const debug = SETTINGS.reactGaDebug === "true"
+if (SETTINGS.gaTrackingID) {
+  ReactGA.initialize(SETTINGS.gaTrackingID, { debug: debug })
+}
+
+class Analytics extends React.Component {
+  componentDidUpdate(prevProps) {
+    const { location } = this.props
+
+    if (location !== prevProps.location) {
+      ReactGA.pageview(window.location.pathname)
```

Comment:
Should we use `location.pathname` here?

Reply:
I went with `window.location.pathname` simply because that's what we were using in MM - do you think there's a reason to go with one or the other? I suppose they _should_ be equal all the time